lms/envs/devstack.py

```diff
@@ -318,7 +318,7 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
 ######################### Django Rest Framework ########################
 
 REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] += (
-    'rest_framework.renderers.BrowsableAPIRenderer',
+    'rest_framework.renderers.JSONRenderer',
```

Comment:
I wouldn't change this here

Reply:
definitely not my intention to commit it.